src/SDKs/CognitiveServices/dataPlane/Vision/CustomVision/Prediction.Tests/Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction.Tests.csproj

```diff
@@ -1,18 +1,16 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.CustomVision.Prediction.Tests</AssemblyName>
-    <Version>1.0.0</Version>    
+    <Version>1.0.0</Version>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFrameworks>net452</TargetFrameworks>
```

Comment:
If you would like to target your test project to .NET 452, please create a separate test project.
We have deliberately remove target framework from the csproj file, so that we can centrally control it.
If you have a need to for Full Desktop test, it should be done via a separate test project.
e.g. https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/Subscription/Subscription.Tests